api/tests/Api/ResetPassword/UpdatePasswordTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Tests\Api\ResetPassword;
+
+use App\Entity\User;
+use App\Security\ReCaptcha\ReCaptcha;
+use App\Tests\Api\ECampApiTestCase;
+use ReCaptcha\Response;
+
+/**
+ * @internal
+ */
+class UpdatePasswordTest extends ECampApiTestCase {
+    // TODO tests for normal operation of password update
+    // TODO extensive tests for authentication checks using the reset password token
+    // TODO tests for recaptcha check
+    // TODO tests for requesting a password reset and retrieving the password reset data, in separate test files
+
+    protected ?User $user = null;
```

Comment:
must this field be nullable?
should we do this in all tests and set the field initially to null?

Reply:
I think it has to be nullable because I don't fill it in the constructor. But I might be wrong. I assigned null because then the error wouldn't be as ugly if I tried to access $this->user before assigning to it. No particularly good reason, I know.